api/extras/.openapi-generator-ignore

```diff
@@ -0,0 +1 @@
+README.md
```

Comment:
Can we add `utils.gen.go` to this list? AFAIK it's a duplicate of the existing `api/utils.gen.go`

Reply:
I think we want to keep it regenerating because the generated code in this repo needs a `utils.gen.go` in its own module (otherwise we'd need to hack the generated code to use the one in the other module)